app/src/main/res/layout/fragment_address_editor.xml

```diff
@@ -306,7 +306,7 @@
             app:hintEnabled="false"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toEndOf="@+id/state_layout"
-            app:layout_constraintTop_toBottomOf="@+id/zip_title">
+            app:layout_constraintTop_toBottomOf="@+id/zip_title" >
```

Comment:
nit don't think we need this or am i missing something? 🧐

Reply:
I'll revert both of these. Was trying to stay consistent with the tags that are closed inline, but I am now seeing that tags that are not closed inline typically don't have the space.